docs/Project-Configuration/Github-Integrations.md

```diff
@@ -109,7 +112,13 @@ buildvariants:
     tasks:
       - name: "print_apple"
         create_check_run:
-          path_to_outputs: "src/output.json" ## for no output use ""
+          path_to_outputs: "src/output.json" ## this file will be checked after the task finishes running
+      - name: "valid_check_run"
```

Comment:
since these are all valid, can we rename it something else like "check_run_no_output" and "create_check_run_with_output" for the one above?